x-pack/plugins/security_solution/server/search_strategy/timeline/factory/events/all/helpers.test.ts

```diff
@@ -320,6 +320,7 @@ describe('#formatTimelineData', () => {
           signal: {
             original_time: ['2021-01-09T13:39:32.595Z'],
             status: ['open'],
+            threshold_result: ['{"count":10000,"value":"2a990c11-f61b-4c8e-b210-da2574e9f9db"}'],
```

Comment:
this looks very different from the `threshold_result` object that's inserted into the signals [here](https://github.com/elastic/kibana/pull/90826/files#diff-8dca2da82db2ec8d439d21b2741ef0ee405dc5a28d410ddf704d960c1c80149fR184)

Reply:
See above.